Dockerfile

```
@@ -0,0 +1,18 @@
+FROM node:12.16.2 AS builder
+
+WORKDIR /usr/build
+RUN npm i npm@latest -g
+
+COPY . /usr/build
+RUN npm install
+RUN node ace build --production
+
+FROM node:12.16.2-alpine3.11 AS app
+
+WORKDIR /usr/src/app
```

```suggestion
WORKDIR /usr/src/app

ENV NODE_ENV production
ENV ENV_SILENT true
```

ENV_SILENT is to avoid Adonis throwing an error if there is no `.env` file in the project's directory. Usually with docker-compose we directly set the environment variables in the container from the `.env` file and the file is never seen by Adonis.